packages/reframed/src/reframed.ts

```diff
@@ -392,97 +404,112 @@ function monkeyPatchIFrameDocument(
 				);
 			},
 		});
+	}
```

Comment:
Could someone double check if this change is needed. I believe we were patching the history functions twice because we were doing it in this loop:

https://github.com/web-fragments/web-fragments/pull/34/files#diff-5bd72dce661e784e4d04596ba00610b72b1cea81099c38f39b9d54bf1687a591R391

Was that intentional?